src/components/ec-filter-popover/ec-filter-popover.story.js

```diff
@@ -0,0 +1,115 @@
+import { text } from '@storybook/addon-knobs';
+import EcFilterPopover from './ec-filter-popover.vue';
+import EcCheckbox from '../ec-checkbox/ec-checkbox.vue';
+
+export default { title: 'Filter Popover' };
```

Comment:
Can we create a separate section for filters, just like we have for layout and CSS? Something like `Filters / Filter Popover`